chirho/counterfactual/ops.py

```diff
@@ -23,10 +25,61 @@ def split(obs: T, acts: Tuple[Intervention[T], ...], **kwargs) -> T:
     return scatter(act_values, event_dim=kwargs.get("event_dim", 0))
 
 
+@pyro.poutine.runtime.effectful(type="undo_split")
```

Comment:
`undo_split` is just a pure function, it doesn't need to be marked `effectful`.

Reply:
Right. Removed.